chore: configure Renovate master issue

rarkins opened the pull request:
Hi @mike-north, I saw you did a lot of manual rebasing just now and thought you might like to try out a new feature: Renovate Master Issue. If configured, it will open a single issue in the repo that lists every PR both pending and open, and provides single click to rebase any of those.

mike-north commented:
@rarkins tbh it's easier to do it with the label since I can select many failing PRs and apply a label to all at once

rarkins commented:
@mike-north interesting workflow. Is that part of the GitHub.com UI or something else?

mike-north commented:
@rarkins github UI 
<img width="1132" alt="screen shot 2018-11-07 at 10 43 50 pm" src="https://user-images.githubusercontent.com/558005/48182034-aabe1080-e2de-11e8-95ba-1f1f21b5b181.png">

rarkins commented:
Thanks for the screenshot. I hadn't thought about the search filter screen.

mike-north commented:
FWIW it's way more useful than the recommendation in the initial PRs (adding `renovate! ` to PR titles). I think this hint (that the `rebase` label was special) was only revealed the first time I had a lockfile issue

rarkins commented:
Adding the label was the first method I added for supporting manual rebase, but then I thought there were plenty of users who wouldn't want to "pollute" their label list with a special one for Renovate, so I added the option for title instead. Based on your feedback it sounds like I should promote this one a little more in the docs. I'll probably revise it this week.
